src/analysis/bounds.rs

```diff
@@ -123,7 +123,10 @@ impl Bounds {
                     if let Some(function) = find_function(env, &finish_func_name) {
                         let mut out_parameters = find_out_parameters(env, function);
                         if use_function_return_for_result(env, &function.ret) {
-                            out_parameters.insert(0, rust_type(env, function.ret.typ).expect("get rust type from return"));
+                            match rust_type(env, function.ret.typ) {
+                                Ok(rt) => out_parameters.insert(0, rt),
+                                Err(_) => return (None, None),
```

Comment:
This way we silently lost that function return something.
Better use `rust_type(...).into_string()`

Reply:
At minimum it worth to try

Reply:
This was mostly intended as a hack to avoid panic for now. It doesn't really
change anything with respect to generated code, because function will be
omitted anyway and return value doesn't matter. Though, now I see that panic
can be easily avoided with config change, so this no longer feels necessary.